pulp_container/app/models.py

```diff
@@ -751,6 +751,8 @@ class ContainerDistribution(Distribution, AutoAddObjPermsMixin):
         null=True,
     )
 
+    max_payload_size = models.IntegerField(null=True)
```

Comment:
I do not think we want to make this parameter configurable. I vote for having a constant for this.